CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
-# 6.6.0 (unreleased)
+# 6.6.0 (2020-01-27)
+
+## Notice about upcoming changes in backendSrv for plugin authors
+
+In our mission to migrate away from AngularJS to React we have removed all AngularJS dependencies in the core data retrieval service `backendSrv`. This change is already in master and will be introduced in the next `major` Grafana release.
```

Comment:
```suggestion
In our mission to migrate away from AngularJS to React we have removed all AngularJS dependencies in the core data retrieval service `backendSrv`.
```

This is information is tied to a specific release of Grafana, it being in master should be self evident.

Reply:
We're just making sure that everyone understands that this is an upcoming change that will be in the next major release.

Reply:
```suggestion
In our mission to migrate away from AngularJS to React we are removing all AngularJS dependencies in the core data retrieval service `backendSrv`. This change will be introduced in Grafana 7.0.0.
```

?

Reply:
(assuming you mean major in the semver kind of way)

Reply:
We're not sure if this is going to be released in 6.7 or 7.0 @sakjur @xlson  so that's why we wrote this open ended text

Reply:
Ah, major release implies `7.0.0` to me (first digit is major, second is minor, and third is patch).

```suggestion
In our mission to migrate away from AngularJS to React we are removing all AngularJS dependencies in the core data retrieval service `backendSrv`. This change will be introduced in the next regular Grafana release.
```

Maybe? And then down below something like

```
You can test your plugin with the `master` branch version of Grafana.
```

Reply:
Ah, in that case I think we should put it under 7.0.0 (unreleased). We can move it to the 6.7.0 section if we end up releasing a 6.7.0 with it.